state_version.go

```diff
@@ -262,7 +262,7 @@ type StateVersionOutputsListOptions struct {
 }
 
 // Outputs retrieves all the outputs of a state version by its ID.
-func (s *stateVersions) Outputs(ctx context.Context, svID string, options StateVersionOutputsListOptions) ([]*StateVersionOutput, error) {
+func (s *stateVersions) Outputs(ctx context.Context, svID string, options StateVersionOutputsListOptions) (*StateVersionOutputsList, error) {
```

Comment:
Perhaps we should make the list option a pointer. This is based off @uturunku1 's work to make all list options pointers #291

Reply:
@sebasslash I'd like to merge it without changing this and update her branch so that all those nil options changes appear in the same release. I'm trying to minimize the method signature changes so we can release this as a minor bump

Reply:
I second the suggested approach, instead of going through our support channels I opened the PR to get this fixed soon so ideally we can avoid any delays - thanks!